internal/app/api/v1/metrics.go

```diff
@@ -39,7 +39,7 @@ func (m Metrics) IncExecution(scriptExecution kubtest.ScriptExecution) {
 	m.Executions.With(map[string]string{
 		"type":   scriptExecution.ScriptType,
 		"name":   scriptExecution.ScriptName,
-		"result": scriptExecution.Execution.Status,
+		"result": string(*scriptExecution.Execution.Status),
```

Comment:
Can we make status required to get rid of this pointers ?

Reply:
yes that makes sense

Reply:
actually - let's do this in a separate PR - there are a couple of other things we should fix in the openapi definition - i'll create an issue for this

Reply:
created #285 for this